aliases/README.md

```diff
@@ -115,3 +116,19 @@ OMB_ALIAS_PACKAGE_MANAGER_SUDO=
 | `tplan`  | `terraform plan`             |
 | `tapply` | `terraform apply`            |
 | `tfmt`   | `terraform fmt`              |
+
+## alias:nala
```

Comment:
Thank you for writing a detailed description for each alias. However, I'm not sure what to do with it.

First of all, all of the section titles `alias:*` are actually the module names, which can be loaded by `_omb_module_require`. However, we don't have a module named `alias:nala`. The `nala` aliases are a part of `alias:package-manager` module. At least, we need to give it a different section name and make it a subsection of `## alias:package-manager`. Then, another problem is that we have detailed descriptions only for the `nala` aliases, which is an imbalance. Although it may require some investigations, could you also add similar descriptions to all the other package-manager aliases?

Reply:
My bad, I'm quite new to all this (git, GitHub, contributing to projects, etc) so I appreciate the patience 😊

As for fixing the documentation, what would be the recommended naming scheme/ template for making it a subsection? Same as for the Debian documentation? (e.g. "# Nala Package Manager" w/ descriptions below)

Should I put it at the bottom below the info about sudo? 

And last but not least, in reference to adding adtl descriptions, I can definitely do that!

Reply:
> Same as for the Debian documentation? (e.g. "# Nala Package Manager" w/ descriptions below)

That is a good one. One thing to add is that the section level should be`###` to make it a subsection of `## alias:package-manager`.

> Should I put it at the bottom below the info about sudo?

No, "making a subsection of `alias:package-manager`" means that make the section of nala aliases *a part* of the section `alias:package-manager`.

Reply:
Sorry about that, I think I've addressed it in my latest push. 😅

If I haven't, could you please eli5 where you'd like it to be? Sorry for the inconveniences.

Reply:
> > Should I put it at the bottom below the info about sudo?
> 
> No, "making a subsection of `alias:package-manager`" means that make the section of nala aliases _a part_ of the section `alias:package-manager`.

Ah, OK. I misunderstood what you mean by the *bottom*. Please forget the above part of my comments.